helperFunctions/subscribeHelpers/SalesforceAgentAssignedHelpers/SubscribeToLiveAgentHelper.ts

```diff
@@ -43,15 +44,17 @@ export class SubscribeToLiveAgent {
 			.then(async (response) => {
 				if (response.statusCode === 403) {
 					console.log(ErrorLogs.LIVEAGENT_SESSION_EXPIRED);
-					await handleEndChatCallback.handleEndChat();
+					this.modify.getAnalytics().sendEvent(getEventData(this.data.room.id, EventName.LIVEAGENT_SESSION_EXPIRED));
+					await handleEndChatCallback.handleEndChat(CloseMessages.LIVEAGENT_SESSION_EXPIRED);
```

Comment:
Can we move this `sendEvent` inside `handleEndChat` function? So that it sends the event when the chat is actually closed. 

Something like to this https://github.com/WideChat/Apps.Salesforce.LiveAgent/blob/master/handlers/InitiateSalesforceSessionHandler.ts#L110...L121